test/fail_compilation/ice14446.d

```diff
@@ -1,14 +0,0 @@
-// REQUIRED_ARGS: -o-
```

Comment:
NOTE: this test is now passing...moved to `test/compilable/ice14446.d`.  It now passes because the compiler now fully supports importing via filename so long as it doesn't break import order invariance.